File: php/filter-content/filtering_get_items_by_linked_item.php
<?php
// Gets a list of items where the 'my_page' item is used in the 'navigation' element.
$items = $client->getItems((new QueryParams())
  ->contains('elements.navigation', 'my_page'));

// Gets items linked to at least Jane, John, or both.
$items = $client->getItems((new QueryParams())
  ->any('elements.author', ['jane_doe','john_wick']));
?>

<?php
// Gets pages linking travel insurance as their subpage.
$items = $client->getItems((new QueryParams())
  ->contains('elements.subpages', 'travel_insurance'));

// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
$items = $client->getItems((new QueryParams())
  ->any('elements.subpages', ['travel_insurance','car_insurance']));
?>

File: java/filter-content/filtering_get_items_by_linked_item.java
// Gets a list of items where the 'my_page' item is used in the 'navigation' element.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterContains("elements.navigation", "my_page")
    .build()
);


// Gets items linked to at least Jane, John, or both.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterAny("elements.author", "jane_doe", "john_wick")
    .build()
);

// Gets pages linking travel insurance as their subpage.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterContains("elements.subpages", "travel_insurance")
    .build()
);

// Gets pages linking at least travel insurance, car insurance, or both as their subpage.
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterAny("elements.subpages", "travel_insurance", "car_insurance")
    .build()
);
